pod/meeting/urls.py

```diff
@@ -18,6 +18,20 @@
     ),
     path("end/<slug:meeting_id>/", views.end, name="end"),
     path("end_callback/<slug:meeting_id>/", views.end_callback, name="end_callback"),
+
+]
+
+if not views.MEETING_DISABLE_RECORD:
+    urlpatterns += [
+        path("recordings/<slug:meeting_id>/", views.recordings, name="recordings"),
+        path(
+            "delete_recording/<slug:meeting_id>/<slug:recording_id>/",
+            views.delete_recording,
+            name="delete_recording"
+        )
+    ]
+
+urlpatterns += [
     path("<slug:meeting_id>/", views.join, name="join"),
-    path("<slug:meeting_id>/<slug:direct_access>", views.join, name="join"),
+    path("<slug:meeting_id>/<slug:direct_access>", views.join, name="join")
```

Comment:
c'était bien avec la virgule à la fin, non  (pareil ligne 30) ?
J'avais vu ça dans les bonnes pratiques 
https://peps.python.org/pep-0008/#when-to-use-trailing-commas
C'est surtout que quand on ajoute un élément, c'est 'plus propre' dans l'historique git

Reply:
je crois que de toute façon c'est ensuite automatiquement corrigé par black ca non ?